contrib/internal/sqltest/sqltest.go

```diff
@@ -49,6 +49,8 @@ func RunAll(t *testing.T, cfg *Config) {
 	cfg.mockTracer = mocktracer.Start()
 	defer cfg.mockTracer.Stop()
 
+	// Make sure testConnect runs first to ensure a connection is established
```

Comment:
I'm a bit confused here. Why does this need to run first? Is it because if you run the other tests you will get an additional span for `Connect` and it will be a bit more difficult to test? In this case, knowing that you get it? In that case, the comment is a bit confusing, perhaps something like:
```suggestion
	// Run this first because the initial DB call generates the "Connect" span.
```

Reply:
Ahh, So this was from the original PR and at first I didn't know why. Turns out that the tests were relying on the fact that the pooled db connection was being reused (so additional "connect" spans weren't being created for the other tests). However, adding connect spans caused a bunch of other tests including the `gorm` and `sqlx` libraries to break as they weren't expecting the new `connect` spans.  By forcing the DB to not maintain any idle connections I was able to force all the tests to always create a new connection in a repeatable way. 

The short answer is: this test doesn't need to be run first separately anymore so I've moved it inside the loop with the other tests.